docs/product-portal/applications/consuming-megazord-libraries.md

```diff
@@ -91,11 +91,12 @@ to fit the needs of existing Firefox applications:
 | `lockbox` | `fxaclient`, `logins` | `org.mozilla.appservices:lockbox-megazord` |
 | `fenix` | `fxaclient`, `logins`, `places` | `org.mozilla.appservices:fenix-megazord` |
 
-Then, simply use gradle's builtin support for [module replacement](https://docs.gradle.org/current/userguide/customizing_dependency_resolution_behavior.html#sec:module_replacement)
+Then, use gradle's builtin support for [module replacement](https://docs.gradle.org/current/userguide/customizing_dependency_resolution_behavior.html#sec:module_replacement)
```

Comment:
Gradle is many things, but "simple" ain't one, at least not in the way we're using it here...